### PR TITLE
Maintenance updates on schema, dependencies, and runtime versions.

### PR DESCRIPTION
**Pull request by mrinaudo-aws**
*Issue #, if available:*

*Description of changes:*
Using version 14 of node, updated dependencies, and schema.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: resource-types/typescript-example-website-monitor/example-monitoring-website.json [hunk 1]
```diff
@@ -102,6 +102,9 @@
         "/properties/Status",
         "/properties/SlaThreshold"
     ],
+    "writeOnlyProperties": [
+        "/properties/ApiKey"
```
Comment (joshuadeanhall) [inline resource-types/typescript-example-website-monitor/example-monitoring-website.json:106]: We may want to consider updating this to use `Type Configuration`.

File: resource-types/typescript-example-website-monitor/src/models.ts [hunk 2]
```diff
@@ -137,3 +137,10 @@ export class ResourceModel extends BaseModel {
     }
 }
 
+export class TypeConfigurationModel extends BaseModel {
+    ['constructor']: typeof TypeConfigurationModel;
+
```
Comment (joshuadeanhall) [inline resource-types/typescript-example-website-monitor/src/models.ts:142]: nit: Looks like extra spaces were added here.

**Review by joshuadeanhall — APPROVED** [review]

**Review by wkexiang — APPROVED** [review]

File: resource-types/typescript-example-website-monitor/src/models.ts (hunk 2 above)
Comment (mrinaudo-aws) [reply resource-types/typescript-example-website-monitor/src/models.ts:142]: This file is auto-generated by the cfn cli.

File: resource-types/typescript-example-website-monitor/example-monitoring-website.json (hunk 1 above)
Comment (mrinaudo-aws) [reply resource-types/typescript-example-website-monitor/example-monitoring-website.json:106]: I see this [post](https://aws.amazon.com/blogs/mt/introducing-typescript-support-for-building-aws-cloudformation-resource-types/) references this resource with a walkthrough and also, as part of consuming the resource, I see the dynamic reference method that uses `ApiKey: "{{resolve:secretsmanager:typescript-example/apikey:SecretString}}"`; we might want to leave it as it is.

**Review by reza-aws — APPROVED** [review]

**Pull request merged**